DigitalLearningSolutions.Web/Views/LearningPortal/Completed/Completed.cshtml

```diff
@@ -1,4 +1,6 @@
-﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Completed
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.ViewComponents

```

Comment:
I don't think @using DigitalLearningSolutions.Web.ViewComponents is necessary in any of these files

Reply:
Yep, removed them all. Must be something in my settings that keeps importing it.